staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -292,83 +290,100 @@ var ignoredPodSubresources = map[string]bool{
 	"status":      true,
 }
 
-func (a *Admission) ValidatePod(ctx context.Context, attrs Attributes) *admissionv1.AdmissionResponse {
+func (a *Admission) ValidatePod(ctx context.Context, attrs api.Attributes) *admissionv1.AdmissionResponse {
 	// short-circuit on ignored subresources
 	if ignoredPodSubresources[attrs.GetSubresource()] {
+		a.RecordEvaluation(a.defaultPolicy, metrics.DecisionAllow, attrs)
 		return allowedResponse()
 	}
 	// short-circuit on exempt namespaces and users
 	if a.exemptNamespace(attrs.GetNamespace()) || a.exemptUser(attrs.GetUserName()) {
+		a.RecordEvaluation(a.defaultPolicy, metrics.DecisionExempt, attrs)
```

Comment:
I agree with that ` just omit the policy labels in this case (and the error cases below) and add separate metrics for tracking exempt requests & errors` . I think it will make the metric more readable.😳